src/sass/shared/skeleton.scss

```diff
@@ -7,3 +7,27 @@ body {
 h1, h2, h3, h4, h5, h6 {
     font-family: $font-titles;
 }
+
+.band {
+    padding: 90px 0 90px;
+    width: 100%;
+
+    &_grey {
+        background: $color-main-bg;
+    }
+
+    & > &__container {
+        @include container();
+
+        & > {
+
+            h1 {
+                @include default-title-big();
+            }
+
+            h2 {
+                @include default-title-medium();
```

Comment:
This rule apply at "skeleton leve": only for `h1` and `h2` 1st level inside the the `band__container`.